trompace/mutations/musiccomposition.py

```diff
@@ -0,0 +1,275 @@
+  ) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+  }}'''
+
+REMOVE_MUSIC_COMPOSITION_BROAD_MATCH = '''RemoveMusicCompositionBroadMatch(
+    from: {{identifier: "{identifier_1}" }}
+    to: {{identifier: "{identifier_2}" }}
+  ) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+  }}'''
+
+ADD_MUSIC_COMPOSITION_WORK_EXAMPLE_COMPOSITION = '''AddDMusicCompositionExampleOfWork(
+    from: {{identifier: "{identifier_1}"}}
+    to: {{identifier: "{identifier_2}"}}
+) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+}}'''
+
+MERGE_MUSIC_COMPOSITION_WORK_EXAMPLE_COMPOSITION = '''MergeMusicCompositionExampleOfWork(
+    from: {{identifier: "{identifier_1}"}}
+    to: {{identifier: "{identifier_2}"}}
+) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+}}'''
+
+REMOVE_MUSIC_COMPOSITION_WORK_EXAMPLE_COMPOSITION = '''RemoveMusicCompositionExampleOfWork(
+    from: {{identifier: "{identifier_1}"}}
+    to: {{identifier: "{identifier_2}"}}
+) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+}}'''
+
+CREATE_MUSIC_COMPOSITION = '''CreateMusicComposition(
+        {parameters}
+  ) {{
+    identifier
+  }}'''
+
+UPDATE_MUSIC_COMPOSITION = '''UpdateMusicComposition(
+        {parameters}
+) {{
+  identifier
+}}'''
+
+DELETE_MUSIC_COMPOSITION = '''DeleteMusicComposition(
+    {parameters}
+  ) {{
+    identifier
+  }}'''
+
+ADD_MUSIC_COMPOSITION_TO_CONTROL_ACTION_MUTATION = """AddActionInterfaceThingInterface (
+            from: {{identifier: "{identifier_2}", type: ControlAction}}
+            to: {{identifier: "{identifier_1}", type: MusicComposition}}
+            field: result
+        ) {{
+            from {{
+                __typename
+            }}
+            to {{
+                __typename
+            }}
+        }}"""
+
+
+def mutation_create_music_composition(title: str, contributor: str, creator: str, source: str,
+                           language: str, inLanguage:str, formatin:str="text/html", name: str=None, description: str=None):
+    """Returns a mutation for creating a music composition object
+    Arguments:
+        title: The title of the page from which the music composition information was extracted.      
+        contributor: A person, an organization, or a service responsible for contributing the music composition to the web resource. This can be either a name or a base URL.
+        creator: The person, organization or service who created the thing the web resource is about.
+        source: The URL of the web resource to be represented by the node.
+        language: The language the metadata is written in. Currently supported languages are en,es,ca,nl,de,fr
+        inLanguage: The language of the music composition. Currently supported languages are en,es,ca,nl,de,fr
+        formatin: A MimeType of the format of the music composition, default is "text/html"
```

Comment:
This is not quite right. It should be the format of the _page_ that describes this composition. 
Why is this called formatin, and not format?

Reply:
"format" is a reserved word for a built-in python function

Reply:
good point. To clarify this though, format is not a reserved keyword (like 'in' or 'class' or 'return'), it's just a function that exists in the default namespace. It wouldn't be a problem to use it (e.g. I often use 'id' as a field even though it's an existing function).
Looking at some guidelines, it looks like the normal suggestion with these variables is to add a `_` after the name: https://softwareengineering.stackexchange.com/a/339309
I recommend that we do the same